src/store.js

```diff
@@ -0,0 +1,21 @@
+import { createStore } from 'redux'
+// import applyMiddleware from "redux/src/applyMiddleware";
+
+import reducer from './reducers'
```

Comment:
this is just an alias, but `reducers` is more descriptive than `reducer`, as they are a collection of reducers not one.

Reply:
Yes, Your are right.